proxy/proxy.go

```diff
@@ -0,0 +1,47 @@
+package proxy
+
+import (
+	"fmt"
+
+	"github.com/spec-tacles/go/broker"
+	"github.com/streadway/amqp"
+	"github.com/vmihailenco/msgpack"
+)
+
+type Proxy struct {
+	Broker    *broker.AMQP
+	RestEvent string
+	Token     string
+}
+
+func (p *Proxy) Make(method string, path string, body *interface{}, options RequestOptions) (*Response, error) {
```

Comment:
Taking a pointer to options here would eliminate a map copy and allow users to pass nil options if they don't want to specify any.